src/array.cr

```diff
@@ -154,6 +185,13 @@ class Array(T)
     ary
   end
 
+  # Returns the number of elements in the array.
+  #
+  # ```
+  # [:foo, :bar].size # => 2
+  # ```
+  getter size : Int32
```

Comment:
Isn't this code also declared earlier in the same file and context? Same comment too.

Reply:
You are right! I always forget to implement an error check in the compiler for this case...